<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ADMIN-PC" name="TestNgorg.timeout" tests="2" failures="0" timestamp="2 Aug 2017 09:33:18 GMT" time="3.013" errors="1">
  <testcase name="A" time="1.011" classname="TestNgorg.timeout">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method org.testng.internal.TestNGMethod.A() didn&#039;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.A() didn't finish within the time-out 1000
]]>
    </error>
  </testcase> <!-- A -->
  <testcase name="B" time="2.002" classname="TestNgorg.timeout"/>
</testsuite> <!-- TestNgorg.timeout -->
